woocommerce/woocommerce-gutenberg-products-block
# Cart i2 Attribute Audit

Audit of cart i2 attributes that:

- Removes top-level attributes if defined in inner blocks
- Passes down hasDarkControls via context
- Moves currentView to Editor Context

Fixes #4828

### Testing

How to test the changes in this Pull Request:

1. Select top level cart i2 block
2. Enable dark mode controls
3. Confirm they are shown in shipping form
4. Confirm they are shown on frontend
5. Test view switcher in cart i2 still works
